release-notes/release-notes57.md

```diff
@@ -38,6 +38,14 @@ For information on the `table_column_resizing` option, see: [Table plugin - `tab
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
+### Changes to notifications and popups when the editor is blocked
+
+Notifications opened with the Notifications API will now remain visible when the editor's progress state is set to `true` using the {{ site.productname }} `setProgressState()` API. Additionally, popups within the editor will now be closed while the editor's progress state is set to `true`.
```

Comment:
considering we had to clarify amongst ourselves what popups were, maybe some examples would be good? not sure. I don't know if/where we use "popups" as a term elsewhere.

Reply:
We did consider adding a "For information about popups see ..." but then realised there was nowhere in the docs that we explained popups as a concept. Do you think we should Jira up adding that (and then updating this release note)?

Reply:
maybe 

> popups within the editor will ...

to 

> popups within the editor (such as thing popup and otherthing popup) will ...

(obviously replacing thing and other thing) (reminder: don't say- eg, ie, or etc)